tt_metal/common/bfloat16.hpp

```diff
@@ -123,6 +123,32 @@ inline std::vector<std::uint32_t> create_arange_vector_of_bfloat16(uint32_t num_
     return vec;
 }
 
+inline std::vector<bfloat16> create_random_vector_of_bfloat16_native(uint32_t num_bytes, float rand_max_float, int seed, float offset = 0.0f) {
+    auto rand_float = std::bind(std::uniform_real_distribution<float>(0, rand_max_float), std::mt19937(seed));
+
+    std::vector<bfloat16> vec(num_bytes/sizeof(bfloat16), 0);
+    for (int i = 0; i < vec.size(); i++) {
+        float num_1_float = rand_float() + offset;
+        vec[i] = bfloat16(num_1_float);
+    }
+    log_info(tt::LogVerif, "Created a random vector of size {}", vec.size());
+    return vec;
+}
+
+inline void print_golden_metalium_vectors(vector<bfloat16>& golden_vec, vector<bfloat16>& result_vec, bool print_vec_val) {
```

Comment:
can we remove this function and its call in the example? I feel like it'd be better to keep the programming example as clean as possible

Reply:
I'm indifferent to remove it, its usage is a handy function to debug if we have low PCC.

Reply:
I can remove its call from the programming examples

Reply:
ok let's do that then

Reply:
can also remove the `bool print_vec_val` + `if` statement in the print function? This seems like unnecessary logic to me

Reply:
done